src/pages/Main/Main.js

```diff
@@ -1,10 +1,44 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import { Products } from './components/Products';
+import { EventImages } from './components/EventImages';
+import './main.scss';
 
 const Main = () => {
+  const [vw, setVw] = useState(100);
+  useEffect(() => {
+    const timer = setInterval(() => {
+      if (vw <= 100 && vw > -100) {
+        setVw(prev => prev - 100);
+      }
+      if (vw <= -100) {
+        setVw(100);
+      }
+    }, 2000);
+    return () => clearInterval(timer);
+  }, [vw]);
```

Comment:
```suggestion
  const [currentImageIndex, setCurrentImageIndex] = useState(0);

  useEffect(() => {
    const timer = setInterval(() => {
      currentImageIndex >= EVENTIMAGES.length - 1
        ? setCurrentImageIndex(0)
        : setCurrentImageIndex(prev => prev + 1);
    }, 2000);

    return () => clearInterval(timer);
  }, [currentImageIndex]);
```

자동 이미지 슬라이드 기능 잘 구현해 주셨는데, 저는 이렇게 구현할 것 같습니다!
해당 이미지 부분에 혹시 이미지가 추가가 된다면 -200, -300 으로 계속 직접 수정을 해야하기 때문에
전체 이미지의 갯수를 이용하는 방법으로 구현하면 조금 더 유지보수하기 좋은 코드가 될 것 같네요!